homeassistant/helpers/helper_config_entry_flow.py

```diff
@@ -337,3 +366,22 @@ def wrapped_entity_config_entry_title(
     if state:
         return state.name or object_id
     return object_id
+
+
+@callback
+def exclude_own_entities(
+    handler: HelperConfigFlowHandler | HelperOptionsFlowHandler,
+    config: dict[str, Any],
```

Comment:
Let's name it more descriptive
```suggestion
    entity_selector_config: dict[str, Any],
```

Reply:
Fixed in cc8a589